front/src/app/todo-list/todo-list.component.ts

```diff
@@ -62,14 +62,45 @@ export class TodoListComponent implements OnInit, OnDestroy {
     this.destroy$.complete();
   }
 
+  stopPropagation(event: Event) {
+    event.stopPropagation();
+  }
+
+  onRenameTaskGroupInputBlur() {
+    setTimeout(() => {
+      this.selectedTaskGroupIndex = undefined;
+      this.changeDetectorRef.markForCheck();
+    }, 150)

```

Comment:
Leave a comment why `setTimeout` is used here (it's not obvious from the code itself)